ds3-autogen-c/src/main/resources/templates/TypeEnumConstant.tmplt

```diff
@@ -0,0 +1,7 @@
+<#include "CopyrightHeader.tmplt"/>
+
+typedef enum {
+<#list enumConstants as enum>
+    ${enum.getName()},
```

Comment:
http://stackoverflow.com/questions/792753/is-the-last-comma-in-c-enum-required
the comma on the last element is OK for C99+